lib/api/protocol.js

```diff
@@ -440,6 +440,33 @@ module.exports = function(Nightwatch) {
   };
 
   /**
+   * Click at the current mouse coordinates (set by moveto).
+   *
+   * The button can be (0, 1, 2) or ('left', 'middle', 'right'). It defaults to left mouse button, and if you don't pass in a button but do pass in a callback, it will handle it correctly.
+   *
+   * @link /session/:sessionId/click
+   * @param {string|number} button The mouse button
+   * @param {function} [callback] Optional callback function to be called when the command finishes.
+   * @api protocol
+   */
+  Actions.mouseButtonClick = function(button, callback) {
+    var buttonIndex;
+    if (typeof button === 'string') {
+      buttonIndex = [
+        MOUSE_BUTTON_LEFT,
+        MOUSE_BUTTON_MIDDLE,
+        MOUSE_BUTTON_RIGHT
+      ].indexOf(button.toLowerCase());
+
+      if (buttonIndex !== -1) {
+        button = buttonIndex;
+      }
+    }
+
+    return postRequest('/click', {button: button}, callback);
```

Comment:
But you alreaday can !!! please test for yourself...

`.mouseButtonClick()` => `.mouseButtonClick(null)` => `{button: null}`

if you send `{button: null}`, selenium set 'left' by default:
https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/click
> "JSON Parameters:
button - {number} Which button, enum: {LEFT = 0, MIDDLE = 1 , RIGHT = 2}. Defaults to the left mouse button if not specified."

Then my code is completely consistent with your `mouseButtonHandler()` function ;-)